crates/mun/test/main.mun

```diff
@@ -1,8 +1,7 @@
 // function to add two floats
-fn add(a:float, b:float):float {
-  let c = a+b+5.0
-  c
-}
+//fn add(a:int, b:int):int {
```

Comment:
Can this be removed?

Reply:
In my opinion this entire file is evil. I use it to run `mun build` every now and then but it really is my personal hack file.